src/app/shared/entry/info-tab-topic/edit-topic/edit-topic-dialog.component.html

```diff
@@ -20,15 +20,20 @@ <h1 mat-dialog-title>Select a Topic</h1>
             <div>
               <span class="bubble mr-2">
                 <mat-icon *ngIf="topicOption.type === TopicSelectionEnum.MANUAL" class="mat-icon-sm">edit</mat-icon>
-                <mat-icon *ngIf="topicOption.type === TopicSelectionEnum.AUTOMATIC" class="mat-icon-sm">sync</mat-icon>
                 <img
-                  *ngIf="topicOption.type === TopicSelectionEnum.AI"
+                  *ngIf="topicOption.type === TopicSelectionEnum.AUTOMATIC"
                   class="site-icons-small"
+                  src="../assets/images/thirdparty/github.svg"
+                  alt="GitHub icon"
+                />
+                <img
+                  *ngIf="topicOption.type === TopicSelectionEnum.AI"
+                  class="site-icons-small mr-1"
                   src="../../../assets/svg/ai-icon.svg"
                   alt="AI generation icon"
                 />
               </span>
-              <strong>{{ topicOption.label }}</strong>
+              <strong>{{ topicOption.type === TopicSelectionEnum.AUTOMATIC ? 'GitHub' : topicOption.label }}</strong>
```

Comment:
You can change the label for automatic topic here so you don't have to modify this line:

https://github.com/dockstore/dockstore-ui2/blob/e6b57842f3f0eef101885712acc59d3cd9259b92/src/app/shared/entry/info-tab-topic/edit-topic/edit-topic-dialog.component.ts#L105